README.md

```diff
@@ -81,7 +81,7 @@ post '/confirm' do
 end
 ```
 
-See the relevant documentation in our Node, Go, Python or Ruby SDK.
+See the relevant documentation in our (Node)[https://github.com/workos-inc/workos-node], (Go)[https://github.com/workos-inc/workos-go], (Python)[https://github.com/workos-inc/workos-python] or (Ruby)[https://github.com/workos-inc/workos-ruby] SDK.
```

Comment:
Do you think people coming to view the ruby idp link example would care about the other SDKs?

Reply:
i think probably this should just be a generic example, not ruby specific. it just happens to use ruby as the server? hows that sound?

Reply:
If that's the intent then it's fine. I wasn't aware of what we're planning on doing with examples for idp link and whether or not it'd be different than our other apps.. but since it's a hosted widget, makes sense to not have one for every language?

Reply:
right yeah like, the majority of the example is HTML / JS. But it does require the final server side call. 

So lets rename the repo to idp-link-example, and then i think id actually like to link to the exact method documentation from here.